main.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from fastapi import FastAPI, Depends
+from sqlalchemy.orm import Session
+from database import SessionLocal
+import models
+from schemas import AuthorCreate, BookCreate
+import crud
+
+app = FastAPI()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+@app.post("/authors/", response_model=AuthorCreate)
+def create_author(author: AuthorCreate, db: Session = Depends(get_db)):
+    return crud.create_author(db=db, author=author)
+
+@app.get("/authors/", response_model=List[AuthorCreate])
+def get_authors(skip: int = 0, limit: int = 10, db: Session = Depends(get_db)):
+    return crud.get_authors(db=db, skip=skip, limit=limit)
+
+@app.post("/books/", response_model=BookCreate)
+def create_book(book: BookCreate, db: Session = Depends(get_db)):
+    return crud.create_book(db=db, book=book)
+
+@app.get("/books/", response_model=List[BookCreate])

```

Comment:
The response model for the `get_books` endpoint should be `List[Book]` instead of `List[BookCreate]` to include the `id` field in the response.